src/main/java/org/jenkinsci/plugins/deployment/HostRecords.java

```diff
@@ -0,0 +1,40 @@
+package org.jenkinsci.plugins.deployment;
+
+import hudson.EnvVars;
+import hudson.model.AbstractBuild;
+import hudson.model.EnvironmentContributingAction;
+import hudson.model.InvisibleAction;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class HostRecords extends InvisibleAction implements EnvironmentContributingAction {
```

Comment:
It makes sense to add some access modifiers and annotations to the class to prevent the misuse.

Examples:
* <code>private final @Nonnull Collection<HostRecord> hostrecords;</code>
* <code>public HostRecords(@Nonnull Collection<HostRecord> hostRecords)</code>
* <code>public @Nonnull Collection<HostRecord> getHostrecords() { </code>